kaldb/src/main/java/com/slack/kaldb/chunk/ReadOnlyChunkImpl.java

```diff
@@ -163,14 +163,20 @@ public static SearchMetadata registerSearchMetadata(
                 snapshotName, cacheSearchContext.hostname),
             snapshotName,
             cacheSearchContext.toUrl());
-    searchMetadataStore.create(metadata).get(TIMEOUT_MS, TimeUnit.MILLISECONDS);
+    searchMetadataStore
```

Comment:
can we document why this doesn't use createSync? I see the timeout is 5s vs the default timeout in createSync being 15. Maybe we can make that clear as to why we need a smaller timeout for this operation?

Reply:
I went back and looked at this, and it's not clear why we use `TIMEOUT_MS` throughout this class. I do see that it appears `DEFAULT_ZK_TIMEOUT_SECS` was added a week later, so this may be just the first implementation?

I added a deprecation notice to this variable, explaining that we should be switching to the sync methods as possible.